lib/backends/sdc/containers.js

```diff
@@ -1676,20 +1577,22 @@ function buildVmPayload(opts, container, callback) {
         });
     }
 
-    /*
-     * "Host" volumes created using -v http[s]://example/file:/container/file
-     * We'll download http[s]://example/file and write it to a new directory in
-     * the zone's dataset but not in the zoneroot, and we'll mount that into the
-     * container at /container/file.
-     */
-    binds = container.Binds
-        || container.HostConfig && container.HostConfig.Binds;
-    if (binds && binds.length > 0) {
-        log.error({host_volumes: binds},
-            'host volumes are not supported');
-        callback(new errors.DockerError(
-            'host volumes are not supported'));
-        return;
+    if (opts.config.experimental_nfs_shared_volumes !== true) {
+       /*
+        * "Host" volumes created using -v http[s]://example/file:/container/file
```

Comment:
This code runs when experimental NFS shared volumes is _not_ enabled, and the goal was to provide the exact same behavior than today in that case. Does that make sense?